src/storage/admin/AdminProcessor.h

```diff
@@ -58,7 +58,6 @@ class TransLeaderProcessor : public BaseProcessor<cpp2::AdminExecResp> {
         if (partMeta.hosts_.size() == 1) {
             LOG(INFO) << "Skip transfer leader of space " << spaceId
```

Comment:
It seems not allowed to balance for the single replica cluster. 

Should we support some warning before the balance be executed?

Reply:
Why forbid it, it is legal